README.md

```diff
@@ -79,7 +82,7 @@ run stacks in parallel and prevent port collisions we need to align port
 allocation.
 
 Therefore the following scheme has been implemented: The port number is encoded
-like this: **ECCDD**
```

Comment:
@tonyfahrion why removing the generic letter encoded form here? You've written `2CC00` instead. Why?

Reply:
we aren't able to differentiate between stores `DD` via port; except if we change the config_local.php for local-dev only. But as far as I can tell, the user should be able to switch stores via the webui or by adding a different hostname, if required.